syncto/__init__.py

```diff
@@ -17,10 +17,15 @@
 AUTHORIZATION_HEADER = 'Authorization'
 CLIENT_STATE_HEADER = 'X-Client-State'
 
+DEFAULT_SETTINGS = {
+    'syncto.cache_hmac_secret': 'This is not a secret',
```

Comment:
I'm not sure we want to have a default value for this. We should rather error out if it's not defined.